### ISSUE TITLE
Connect the "select an account and add an account" front-end to the back-end

### ISSUE THREAD

**Issue by Thykof** [massalabs/thyra-plugin-wallet#200]

**Comment by Thykof** [comment]
> Rules to implement:
> Name can’t contain special characters
> -> on the click, do the check & display error message: “Special characters can’t be used”  /!\ I think that the fact that we can’t use special character is due to a bug that can be solved. If we have time, it would be better to solve the bug and remove this rule.
> Same name can’t be used twice. Prevent user from creating an account with twice the same name.
> -> on the click, do the check and display error message: “This account name already exists”

**Comment by SlnPons** [comment]
Can't access the page: `http://localhost:8080/web-app/account-select`

**Comment by Thykof** [comment]
In the screens, you access to /web-app/welcome. try /web-app/account-select

**Comment by SlnPons** [comment]
@Thykof have you tried on your side? I can't access the URL I've shared which is the same you shared. I'm automatically redirected to /welcome. 
I can do a video or show you directly ...

**Comment by Thykof** [comment]
Maybe because you don't have any account yet, let's see together

**Comment by SlnPons** [comment]
@Thykof so after we discussed, I tested it again and same thing. I can't access the page (but I can see it for like 1 sec and then again the welcome screen). 

Even if I have to no account, this page should appear. So we need to solve that properly, let's check this together ;) 
But here is the user flows and rules to implement: 
1. As a user, when I install and open the wallet for the first time, I want to see this screen followed by the import or the step-by-step process. 


![Image](https://github.com/massalabs/thyra-plugin-wallet/assets/109611779/203d683d-e59e-4f94-bdd0-22d9fdf6f713)

2. As a user, when I open MassaWallet for the second time (ie: not the first time and even if I've got no account), I want to see this page followed by the account dashboard or the creation steps.


![Image](https://github.com/massalabs/thyra-plugin-wallet/assets/109611779/9b82de20-1a64-46cb-9b99-777dc8ddbcd4)
